lib/matplotlib/colorbar.py

```diff
@@ -674,9 +674,9 @@ def _process_values(self, b=None):
 
             b = self.norm.inverse(self._uniform_y(self.cmap.N + 1))
             if self._extend_lower():
-                b[0] = b[0] - 1
+                b[0] = 0.9 * b[0]
```

Comment:
It seems chaning the end values is not needed for LogNorm the plot will look fine without the multiplier. The only 2 other Norm objects that reach this code are: SymLogNorm and PowerNorm. SymLogNorm cannot be inverted, so it won't work in a colorbar like this. PowerNorm has a different problem at the lower end point, I'll look into it some more...